contributing.md

```diff
@@ -0,0 +1,29 @@
+# Contributing to tinylog
+
+[Pull requests](https://github.com/tinylog-org/tinylog/pulls) are always welcome :) This document should help to build tinylog on your local machine. If you had a build problem nevertheless, you would be welcome to raise a [Github issue](https://github.com/tinylog-org/tinylog/issues/new?assignees=&labels=question&template=question.md&title=).
+
+## Downloading JDK 9
+
+A JDK 9 is required to build tinylog 2. Newer JDKs cannot compile legacy code for older Java versions, and older JDKs cannot compile new features for the latest Java versions.
+
+OpenJDK 9 can be downloaded from [adoptopenjdk.net](https://adoptopenjdk.net/releases.html?variant=openjdk9&jvmVariant=hotspot) for Windows, MacOS, and Linux. The downloaded archive only needs to be unpacked. No installation is required.
+
+## Building tinylog
+
+tinylog 2 can be built with Maven 3.5 and later:
+
+```
+mvn clean verify
+```
+
+Before creating a pull request, please always ensure that this Maven command can be executed successfully. It compiles the code, runs JUnit tests, checks the code style via Checkstyle, and analyzes the code with SpotBugs.
+
+## IDE support
+
+In the [configuration folder](./configuration), there is a [code formatter](./configuration/formatter.xml) for Eclipse and [Checkstyle rules](./configuration/checkstyle-rules.xml), which can be used for Eclipse's and IntelliJ's Checkstyle integration for showing Checkstyle violations directly in the IDE. 
```

Comment:
Also works for VS Code :)

Reply:
I have never used VS Code for Java. Does it support the code formatter or Checkstyle rules?

Reply:
Vs code is very extension driven. The Java language support extension supports formatter configurations.

There is also a checkstyle extension, I haven't tried it though.

Check out this link for further information https://code.visualstudio.com/docs/java/java-linting